File: Data Structures & Algorithms/max-consecutive-ones/submission-0.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int r=0,l=0,mc=0,oc=0,s=0;
        while(r<nums.size()){
            if(nums[r]==1){
                oc+=1;
            }
            else{
                oc=0;
            }
            mc=max(mc,oc);
            r++;
        }
        return mc;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        l=ml=0
        nums=sorted(nums)
        for i in range(1,len(nums)):
            if nums[i]-nums[i-1]==1:
                l+=1
            elif nums[i]-nums[i-1]==0:
                continue
            else:
                l=0
            ml=max(ml,l)
        return ml+1